.gitignore

```diff
@@ -29,3 +29,6 @@ explorer.drive
 
 # Logs
 explorer.log
+
+# Supporting files
+supporting_files/
```

Comment:
aren't they referenced in the source code?

Reply:
Fixed